### DISCUSSION TITLE
PoolInterface buffer regression issue in akka-http 10.2.10

### DISCUSSION THREAD

**Discussion by jphelp32** [apache/pekko-http#138] [General]
@jrudolph
can you comment on the likelihood of pekko-http addressing this regression that was introduced in akka-http 10.2.10: https://github.com/akka/akka-http/pull/4132#issuecomment-1238618118
We are currently locked at akka-http 10.2.9 due to this issue. As we're planning our migration to pekko, I'm trying to plan whether to expect a fix in pekko-http, or if we should plan to adjust our configs according to the newer/broken(?) behavior that was introduced in 10.2.10.
thanks

**Comment by pjfanning** [comment]
This should be logged as an issue

**Reply by jphelp32** [reply]
will do PJ, thanks
